backend/.dockerignore

```diff
@@ -18,6 +18,7 @@ LICENSE
 tests/
 *.sqlite3
 postgres/
+.venv
```

Comment:
Why are we adding `.venv` here?

Reply:
https://github.com/pennlabs/office-hours-queue/pull/329/files#diff-b148396bbfff0ab728abd0f4ecccb2d94ee1929bebbfdcb301a1818a7fa8a351